helm/charts/infra/templates/server/configmap.yaml

```diff
@@ -59,6 +59,16 @@ data:
 {{- $users = append $users (dict "name" "connector" "accessKey" $accessKey) }}
 {{- end }}
 
+{{- if .Values.server.config.ui.proxyURL }}
+    ui:
+      proxyURL: {{ .Values.connector.config.ui.proxyURL }}
+{{- end }}
+
+{{- if and (include "ui.enabled" . | eq "true") (not .Values.server.config.ui.proxyURL) }}
```

Comment:
Does this look right? Idea is to set a default if not specified by the user (and if the UI is on)

Reply:
Yes that looks right. A simple way to verify is with helm template:

```
$ helm template helm/charts/infra -s templates/server/configmap.yaml --set ui.enabled=false
```

Or

```
$ helm template helm/charts/infra -s templates/server/configmap.yaml --set server.config.ui.proxyURL=somevalue```